annotation/tests/test_distribution.py

```diff
@@ -21,24 +23,107 @@
 from annotation.distribution.main import (
     DistributionUser,
     choose_validators_users,
+    distribute,
     distribute_tasks_extensively,
     find_equal_files,
     find_small_files,
     find_users_share_loads,
     get_page_number_combinations,
     prepare_response,
     prepare_users,
+    redistribute,
 )
 from annotation.microservice_communication.assets_communication import (
     prepare_files_for_distribution,
 )
-from annotation.models import File, User
+from annotation.microservice_communication.jobs_communication import (
+    JobUpdateException,
+)
+from annotation.models import File, Job, ManualAnnotationTask, User
 from annotation.schemas import (
     FileStatusEnumSchema,
+    JobStatusEnumSchema,
     TaskStatusEnumSchema,
     ValidationSchema,
 )
 
+
+@pytest.fixture
+def mock_session(
+    tasks_for_test_redistribute: Tuple[ManualAnnotationTask, ...]
+):
+    mock_db = Mock(spec=Session, flush=Mock(), rollback=Mock())
+    mock_db.query.return_value.filter.return_value.all.return_value = (
+        tasks_for_test_redistribute
+    )
```

Comment:
you dont really mock a session, this is a specific mock of a query

Reply:
Youre right. But since I need to mock the function flush(), rollback() and the query, I'll keep the mock object as it is. I'll just rename it to mock_db, since it mocks the db object